scripts/src/blockchain/txSender.ts

```diff
@@ -35,10 +35,13 @@ export class TxSender {
 			.setTimeout(0);
 	}
 
-	public async buildCreateAccount(address: Address, startingBalance: number, fee: number, memoText?: string, channel?: Channel): Promise<TransactionBuilder> {
+	public async buildCreateAccount(address: Address, startingBalance: number, fee: number, memo?: string, channel?: Channel): Promise<TransactionBuilder> {
 		const builder = await this.getTransactionBuilder(fee, channel);
 		builder.setTimeout(0);
-		builder.addMemo(memoText ? Memo.text(memoText) : Memo.text(""));
+		if (memo && typeof memo === "string" && memo.length > MEMO_LENGTH) {
```

Comment:
remove the test here, done internally in TransactionBuilder

Reply:
Once you are trying to call Memo.text(memo) with memo longer than 28 characters, than it throws the Stellar error and not the one that explains that the memo should be shorter than 21 (and not 28).